codewof/programming/content/en/fahrenheit-to-celsius/question.md

```diff
@@ -0,0 +1,8 @@
+# Fahrenheit to celsius
+
+Write a function `fahrenheit_to_celsius(temperature)` that takes a temperature in fahrenheit, and **returns** the temperature converted into celsius.
+The formula for converting fahrenheit into celsius is: (Fahrenheit - 32) x 5/9 = Celsius.
```

Comment:
Same as the other temperature one; consider _degrees_ and swapping the order of the formula